knowledge-base/how-to-expand-document-map-tree-by-default-in-html5-viewers.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Expand Document Map tree by default in Html5 Report Viewers
+description: How to expand Document Map tree initially in the Html5 Viewers
+type: how-to
+page_title: Make Bookmark tree expanded by default in Html5 Viewers
```

Comment:
How to Make Bookmark tree expanded by default in Html5 Viewers?

Reply:
I would add here something that Nasko said in your other reviewer request - use consistent technical terms. Document Map tree or Bookmark tree?